src/utils/changelog.ts

```diff
@@ -679,6 +732,14 @@ function formatChangelogEntry(entry: ChangelogEntry): string {
     }
   }
 
+  // Apply blockquote highlighting if requested
+  if (entry.highlight) {
+    text = text
+      .split('\n')
```

Comment:
Hoping the `text` here has normalized new lines (so they are always `\n` and not `\r` or `\r\n` or something)